deployment/README.md

```diff
@@ -79,53 +79,45 @@ The infrastructure is based around multiple environments, for example `alpha`, `
 
 Once you have setup an environment with `$(nix-build -A deployment.david.deploy)` you will probably want to stick to using `$(nix-build -A deployment.david.applyTerraform)` and `$(nix-build -A deployment.david.syncS3)` only, avoiding dealing with the large plutus tutorial.
 
 ### SSH configuration
-Running the terraform scripts will place an ssh config file in your ~/.ssh/config.d directory. This will give you easy ssh access to the servers by setting the jump hosts, usernames, dns names etc but in order for it to work you must include it in your main ssh config. Open or create the file ~/.ssh/config and add the following line at the top `Include config.d/plutus_playground.conf`. You can then test the config by running `ssh nixops.plutus_playground` which should open an ssh on the nixops machine (TODO: rename nixops machine).
+Running the terraform scripts will place an ssh config file in your [~/.ssh/config.d](~/.ssh/config.d) directory. This will give you easy ssh access to the servers by setting the jump hosts, usernames, dns names etc but in order for it to work you must include it in your main ssh config. Open or create the file ~/.ssh/config and add the following line at the top `Include config.d/plutus_playground.conf`. You can then test the config by running `ssh prometheus.plutus_playground` which should open an ssh on the prometheus machine.
 
-This configuration is also vital for Morph to work as it assumes this ssh config so you must get this working before carrying on.
+> **This configuration is also vital for Morph to work as it assumes this ssh config so you must get this working before carrying on.**
 
 ### Morph - Deploying Server Configuration
 Once you have run the terraform scripts you will have an up-to-date environment with EC2 instances running, however these instances won't have the required NixOS configuration yet. In order to configure them we use morph, there is just one command for normal use: `morph deploy ./deployment/morph/default.nix switch`.
 
-It is important to note that this is somewhat stateful in that you must run the terraform scripts beforehand to make sure that both the ssh configuration and the machine definitions (machines.json generated by terraform) are correct. Otherwise morph could try to deploy to incorrect EC2 instances.
+It is important to note that this is somewhat stateful in that you must run the terraform scripts beforehand to make sure that both the ssh configuration and the machine definitions (machines.json generated by terraform) are correct. Otherwise morph could try to deploy to incorrect EC2 instances. Be especially careful when switching between multiple different environments!
+
+Now that things are up and running you should be able to get some basic feedback by looking at [https://env.goguen.monitoring.iohkdev.io/targets](https://env.goguen.monitoring.iohkdev.io/targets) (where `env` is the environment you deployed to).
```

Comment:
Maybe you could change this to `https://<env>.goguen.monitoring.iohkdev.io/targets` ? Rendering this as a link when env is meant to be a placeholder and not a working URL isn't helpful i think.